README.md

```diff
@@ -69,110 +90,96 @@ pip install -i https://test.pypi.org/simple/ taichi-nightly
   - Apple Metal
   - WebAssembly (experiemental)
 
-<a name="win-note">1</a>. On Windows, please install [Microsoft Visual C++ Redistributable](https://aka.ms/vs/16/release/vc_redist.x64.exe) first.
-
-## Your first Taichi program
-
-Here's how you can program a 2D fractal in Taichi:
-
-```py
-# python/taichi/examples/simulation/fractal.py
-
-import taichi as ti
+### Installation
 
-ti.init(arch=ti.gpu)
+Use Python's package installer **pip** to install Taichi:
 
-n = 320
-pixels = ti.field(dtype=float, shape=(n * 2, n))
+```bash
+pip install taichi
+```
 
+> *We also provide a nightly package. Note that nighly packages may crash because they are not fully tested.  We cannot guarantee their validity, and you are at your own risk trying out our latest, untest features. *
 
-@ti.func
-def complex_sqr(z):
-    return ti.Vector([z[0]**2 - z[1]**2, z[1] * z[0] * 2])
+```bash
+pip install -i https://test.pypi.org/simple/ taichi-nightly
+```
 
+### Run your "Hello, world!"
 
-@ti.kernel
-def paint(t: float):
-    for i, j in pixels:  # Parallelized over all pixels
-        c = ti.Vector([-0.8, ti.cos(t) * 0.2])
-        z = ti.Vector([i / n - 1, j / n - 0.5]) * 2
-        iterations = 0
-        while z.norm() < 20 and iterations < 50:
-            z = complex_sqr(z) + c
-            iterations += 1
-        pixels[i, j] = 1 - iterations * 0.02
+See [Hello,world!](https://docs.taichi.graphics/#hello-world).
 
+### Build from source
 
-gui = ti.GUI("Julia Set", res=(n * 2, n))
+If you wish to try our our experimental features or build Taichi for your own environments, see [Developer installation](https://docs.taichi.graphics/lang/articles/contribution/dev_install).
 
-for i in range(1000000):
-    paint(i * 0.03)
-    gui.set_image(pixels)
-    gui.show()
-```
+# Reference
```

Comment:
Just my personal opinion: I still feel like we should put Community & Contributors before the Refernece section. The demos in the headline can have a link that jumps to the ref section.